### PR TITLE
[CS0618] AbstractStage Warning Disable

### PR DESCRIPTION
**Pull request by eaba**
## Checklist

For significant changes, please ensure that the following have been completed (delete if not relevant):

* [x] This change follows the [Akka.NET API Compatibility Guidelines](https://getakka.net/community/contributing/api-changes-compatibility.html).
* [x] This change follows the [Akka.NET Wire Compatibility Guidelines](https://getakka.net/community/contributing/wire-compatibility.html).

### CODE REVIEW

File: src/core/Akka.Streams/Stage/AbstractStage.cs [hunk 1]
```diff
@@ -59,7 +63,9 @@ internal class PushPullGraphLogic<TIn, TOut> : GraphStageLogic, IDetachedContext
         /// <summary>
         /// TBD
         /// </summary>
+#pragma warning disable CS0618 // Type or member is obsolete
         public AbstractStage<TIn, TOut> Stage { get; }
```
Comment (Aaronontheweb) [inline src/core/Akka.Streams/Stage/AbstractStage.cs:67]: What stage are we supposed to replace `AbstractStage` with @eaba ?

**Review by Aaronontheweb — COMMENTED** [review]
Left a question

File: src/core/Akka.Streams/Stage/AbstractStage.cs (hunk 1 above)
Comment (eaba) [reply src/core/Akka.Streams/Stage/AbstractStage.cs:67]: "Please use GraphStage instead. [1.1.2]" @Aaronontheweb

Comment (eaba) [reply src/core/Akka.Streams/Stage/AbstractStage.cs:67]: https://github.com/akkadotnet/akka.net/blob/b688e5cf2e786f582e7e440210c194b6161f8b98/src/core/Akka.Streams/Stage/AbstractStage.cs#L389-L393

Comment (eaba) [reply src/core/Akka.Streams/Stage/AbstractStage.cs:67]: https://github.com/akkadotnet/akka.net/blob/b688e5cf2e786f582e7e440210c194b6161f8b98/src/core/Akka.Streams/Stage/Stage.cs#L33-L36

**Pull request merged**